content/en/database_monitoring/guide/connect_dbm_and_apm.md

```diff
@@ -264,6 +267,29 @@ client.query('SELECT $1::text as message', ['Hello world!'], (err, result) => {
 
 {{% /tab %}}
 
+{{% tab ".NET" %}}
+
+<div class="alert alert-warning">
+This features requires automatic instrumentation to be enabled for your .NET service.
+</div>
+
+Follow the [.NET Framework tracing instructions][1] or the [.NET Core tracing instructions][2] to install the automatic instrumentation package and enable tracing for your service.
+
+Ensure that you are using a supported client library (e.g. `Npgsql`).
+
+Enable the database monitoring propagation feature by setting the following environment variable:
+   - `DD_DBM_PROPAGATION_MODE=full`
+
+For the best user experience ensure the following environment variables are set in your application:
+   - `DD_SERVICE=(application name)`
+   - `DD_ENV=(application environment)`
+   - `DD_VERSION=(application version)`
```

Comment:
Good callout. We probably should have this for all tracers as well.

Reply:
I stole this from the Python tab 😅

Reply:
ah! 🙄 
It was a good thing to steal. We can follow-up later to maybe move this up but we don't have to do this now.